share/spack/gitlab/cloud_pipelines/stacks/e4s-power/spack.yaml

```diff
@@ -210,6 +210,8 @@ spack:
     pipeline-gen:
     - build-job:
         image: ecpe4s/ubuntu20.04-runner-ppc64le:2023-01-01
+        before_script-:
```

Comment:
Can this get moved in to the before script in the top level? We see this behavior on some other runners beyond just power.